typhon/cloudmask/aster.py

```diff
@@ -492,11 +492,13 @@ def read_coordinates(self, channel="1"):
             "TIR": (14, 15)
         }
 
-        latstr = (f"HDF4_SDS:UNKNOWN:{self.filename}:{sdsidx[self.channel2sensor[channel]][0]}")
+        latstr = (f"HDF4_SDS:UNKNOWN:{self.filename}:"
+                  f"{sdsidx[self.channel2sensor[channel]][0]}")
```

Comment:
```suggestion
        latstr = ":".join(
            (
                "HDF4_SDS",
                "UNKNOWN",
                f"{self.filename}",
                f"{sdsidx[self.channel2sensor[channel]][0]}",
            )
        )
```
I know that you are not a fan of many lines, so this is merely a suggestion 😉